hr_attendance_calendar/views/hr_attendance_view.xml

```diff
@@ -20,26 +20,6 @@
         </field>
     </record>
 
-
-    <record model="ir.ui.view"
```

Comment:
Why is this view removed?

Reply:
It is a duplicate. There is exactly the same view with the same id just above it.